DigitalOffice/Services/TimeManagementService/Models/ChangeWorkTimeRequest.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace LT.DigitalOffice.TimeManagementService.Models
+{
+    public class ChangeWorkTimeRequest
+    {
+        public Guid Id { get; set; }
+        public Guid WorkerUserId { get; set; }
+        public DateTime StartTime { get; set; }
+        public DateTime EndTime { get; set; }
+        public string Title { get; set; }
+        public Guid ProjectId { get; set; }
```

Comment:
Change to "Guid?"

Reply:
I can not understand, why

Reply:
Because it's not good to force the frontend when one variable changes to send the state of all old ones (IMHO).